contrib/endpoints/src/api_manager/mixer/mixer.cc

```diff
@@ -199,13 +202,13 @@ void Mixer::Check(
 
   ::istio::mixer::v1::CheckRequest request;
   request.set_request_index(++request_index_);
-  CovertToPb(info, request.mutable_attribute_update());
+  CovertToPb(info, service_name(), request.mutable_attribute_update());
```

Comment:
no.  service_name_ is a private.   Mixer::Check() should be able to use it directly.